frontend/src/container/Header.jsx

```diff
@@ -52,6 +57,7 @@ class Header extends React.Component {
     const { props } = this
 
     if (props.user.user_id === -1) {
+      Cookies.set(COOKIE_FRONTEND.DEFAULT_LANGUAGE, idLang, {expires: 180})
```

Comment:
the expire time 180 should be in a constant, `COOKIE_FRONTEND.DEFAULT_EXPIRE_TIME` for exemple

Reply:
agreed

Reply:
fixed